--- Startup.cs	2020-06-01 14:46:49.471215058 -0700
+++ StartupForwarded.cs	2020-06-01 14:52:30.056668367 -0700
@@ -14,6 +14,7 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.AspNetCore.HttpOverrides;
 
 namespace BlazorEditor
 {
@@ -71,6 +72,11 @@
                 app.UseHsts();
             }
 
+            app.UseForwardedHeaders(new ForwardedHeadersOptions
+            {
+                ForwardedHeaders = ForwardedHeaders.XForwardedFor | ForwardedHeaders.XForwardedProto
+            });
+
             app.UseHttpsRedirection();
             app.UseRouting();
 
@@ -87,4 +93,4 @@
             });
         }
     }
-}
\ No newline at end of file
+}
